sgkit_vcf/vcf_reader.py

```diff
@@ -166,13 +166,17 @@ def vcf_to_zarr_parallel(
     regions: Union[None, Sequence[str], Sequence[Optional[Sequence[str]]]],
     chunk_length: int = 10_000,
     chunk_width: int = 1_000,
+    temp_chunk_length: Optional[int] = None,
     tempdir: Optional[Path] = None,
 ) -> None:
     """Convert specified regions of one or more VCF files to zarr files, then concat, rechunk, write to zarr"""
 
+    if temp_chunk_length is None:
+        temp_chunk_length = chunk_length
+
     with tempfile.TemporaryDirectory(prefix="vcf_to_zarr_", dir=tempdir) as tmpdir:
 
-        paths = vcf_to_zarrs(input, tmpdir, regions, chunk_length, chunk_width)
+        paths = vcf_to_zarrs(input, tmpdir, regions, temp_chunk_length, chunk_width)
```

Comment:
Won't this cause the size of the blocks in memory to be `temp_chunk_length` x all samples?

Reply:
An nm, of course it will (was interpreting that incorrectly).  

Thoughts on enforcing that it evenly divide `chunk_length` (or suggesting as much in the docs)?

Reply:
Yes, I think that's a good idea. Fixed in https://github.com/pystatgen/sgkit-vcf/pull/8/commits/3e8fa105b8e91ddaebfa6975ca7affabc72ccfe5